Lib/urllib/parse.py

```diff
@@ -366,9 +366,17 @@ def _fix_result_transcoding():
 def urlparse(url, scheme='', allow_fragments=True):
     """Parse a URL into 6 components:
     <scheme>://<netloc>/<path>;<params>?<query>#<fragment>
-    Return a 6-tuple: (scheme, netloc, path, params, query, fragment).
-    Note that we don't break the components up in smaller bits
-    (e.g. netloc is a single string) and we don't expand % escapes."""
+
+    The result is a ParseResult/SplitResult object, which is a named 6-tuple with
```

Comment:
This will actually be a `ParseResult` or a `ParseResultBytes` object, depending on the type of the `url` parameter. It will never be a `SplitResult` object.